serializations.md

```diff
@@ -31,7 +31,7 @@ constructed directly as a byte array.
 | pow_evidence | 168  |
 | nonce        | 8    |
 | time         | 8    |
-
```

Comment:
I am a little bit divided on that. On one hand it follows the Ethereum pattern, on the other there is more than one beneficiary (because we split the computed value 60-40).

Reply:
Moved to miner.